applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_prebuckling_analysis.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing Kratos
+import KratosMultiphysics
+# Importing the base class
+from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_analysis import StructuralMechanicsAnalysis
+
+class StructuralMechanicsPrebucklingAnalysis(StructuralMechanicsAnalysis):
+    def __init__(self, model, project_parameters):
+        super(StructuralMechanicsPrebucklingAnalysis, self).__init__(model, project_parameters)
+
+    def Initialize(self):
```

Comment:
TBH I still don't see why you need to override it completely
the things you do in here you could IMO also do with your own initialize like this:
~~~py
def Initialize(self):
    super().Initialize()
    if problem_data.Has("start_time"):
    # ... and so on
~~~

Reply:
Well, that makes a lot of sense I am afraid. Will do the changes.

Reply:
Ah now I remember. The super.Initialze() requires start_time and end_time to be defined. However, they are not needed for the buckling solver.

Reply:
hm
maybe then sth like this:
~~~py
def Initialize(self):
    # set dummy start_time and end_time
    super().Initialize()
    # remove dummy start_time and end_time
    if problem_data.Has("start_time"):
    # ... and so on
~~~

this might be a bit hacky but then you could use it a is

I took not that this should be improved in the future in the `AnalysisStage`. After that is done there will be no more need for your hack solution

Reply:
Makes sense. Pushed the changes!